### DISCUSSION TITLE
Example with SingleTickerProviderStateMixin and view models

### DISCUSSION THREAD

**Discussion by hjJunior** [FilledStacks/stacked#1078] [Q&A]
I currently have a `Tabs` widget and to use tabs is required to pass a TickerProvider to vsync, I'm not sure how to do that with view models, would be nice if we have a example

Currently I'm using the DefaultTabController

**Comment by FilledStacks** [comment]
Hey, you wouldn't add it to ViewModels, so if it's in your View then that's fine.
